crates/esthri/src/lib.rs

```diff
@@ -328,3 +319,77 @@ where
 
     Ok(listing)
 }
+
+pub mod opts {
+    use derive_builder::Builder;
+    use glob::Pattern;
+
+    use crate::rusoto::*;
+    #[derive(Debug, Copy, Clone, Builder)]
+    pub struct AwsCopyOptParams {
+        #[builder(default = "Some(S3StorageClass::Standard)")]
+        pub storage_class: Option<S3StorageClass>,
```

Comment:
This shouldn't be an option anymore if it has a default

Reply:
Removing the Option wrapper should also simplify the usage of this field.

Reply:
The reason Option wrapper is used for storageclass is because of this line:
https://github.com/swift-nav/esthri/blob/e29f684b5a618bab6d0a23682ec931444100f28d/crates/esthri/src/ops/upload.rs#L129

If storageclass is not set then we are taking the value from the env variable. Using Option wrapper would allow it to use unwrap_or_else() so that we would potentially write less boilerplate sin the upload and cli crates for matching different storageclass value

Reply:
Missed that, thanks